packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketSelectedOfferVerifyOptions.tsx

```diff
@@ -53,104 +54,120 @@ const AccountType = styled.span`
-                                                ? option.account.accountType
-                                                : ''}
-                                        </AccountType>
-                                    </AccountName>
-                                    <Amount>
-                                        <CryptoWrapper>
-                                            <FormattedCryptoAmount
-                                                value={formattedBalance}
-                                                symbol={symbol}
-                                            />
-                                        </CryptoWrapper>
-                                        •
-                                        <FiatWrapper>
-                                            <FiatValue amount={formattedBalance} symbol={symbol} />
-                                        </FiatWrapper>
-                                    </Amount>
-                                </AccountWrapper>
-                            </Option>
-                        );
+                            return (
+                                <Option>
+                                    <LogoWrapper>
+                                        <CoinLogo size={25} symbol={symbol} />
+                                    </LogoWrapper>
+                                    <AccountWrapper>
+                                        <AccountName>
+                                            <AccountLabeling account={option.account} />
+                                            <AccountType>
+                                                {option.account.accountType !== 'normal'
+                                                    ? option.account.accountType
+                                                    : ''}
+                                            </AccountType>
+                                        </AccountName>
+                                        <Amount>
+                                            <CryptoWrapper>
+                                                <FormattedCryptoAmount
+                                                    value={formattedBalance}
+                                                    symbol={symbol}
+                                                />
+                                            </CryptoWrapper>
+                                            •
+                                            <FiatWrapper>
+                                                <FiatValue
+                                                    amount={formattedBalance}
+                                                    symbol={symbol}
+                                                />
+                                            </FiatWrapper>
+                                        </Amount>
+                                    </AccountWrapper>
+                                </Option>
```

Comment:
Could you please create new components from these switch cases? 🙏 I think it's easier to read it.